src/lib/proof-system/proof.ts

```diff
@@ -92,6 +92,13 @@ class ProofBase<Input = any, Output = any> {
   publicFields() {
     return (this.constructor as typeof ProofBase).publicFields(this);
   }
+
+  static _proofFromBase64(proofString: string, maxProofsVerified: 0 | 1 | 2) {
```

Comment:
In the existing `fromJSON` implementation, we await `initializeBindings`.  What are the implications of skipping that step before entering pickles?